### ISSUE TITLE
/var/log/btmp grows forever

### ISSUE THREAD

**Issue by mischief** [coreos/bugs#395]
something should be done to prevent btmp from growing too large.

```
core@core1 ~ $ ls -lahst /var/log
total 796M
796M -rw-------  1 root utmp            796M Jun 25 22:53 btmp
4.0K -rw-r--r--  1 root root            143K Jun 25 22:51 lastlog
124K -rw-rw-r--  1 root utmp            118K Jun 25 22:51 wtmp
4.0K drwxr-xr-x 10 root root            4.0K Apr 21 09:19 ..
4.0K drwxr-xr-x  3 root root            4.0K Apr 19 10:07 .
   0 -rw-r--r--  1 root root               0 Apr 19 10:07 faillog
4.0K drwxr-sr-x  3 root systemd-journal 4.0K Apr 19 10:07 journal
```

**Comment by mischief** [comment]
this file is created by `systemd/tmpfiles.d/var.conf`. maybe we can remove it, since sshd failures are logged to journald anyway. if the file doesn't exist, it won't be appended to or recreated.

**Comment by mischief** [comment]
fixed by coreos/coreos-overlay#1317
